core/templates/domain/improvements/NeedsGuidingResponsesTaskObjectFactorySpec.ts

```diff
@@ -130,7 +130,9 @@ describe('Needs guiding responses task', function() {
         entity_type: 'exploration',
         entity_id: 'eid',
         entity_version: 1,
-        // @ts-ignore Suppress compile-time error for testing.
+        // The task type should be of type needs_guiding_responses but we set
```

Comment:
Please make this clearer and follow conventions elsewhere. If you put quotes around the type elsewhere, do so here too. All the cases in these stats-related files should have very similar comments.

Reply:
done